### DISCUSSION TITLE
Small UI improvements

### DISCUSSION THREAD

**Discussion by bsodmike** [orhun/gpg-tui#234] [Show and tell]
![Screenshot 2024-07-07 at 14 44 51](https://github.com/orhun/gpg-tui/assets/368260/7c6443a8-8a82-4760-958b-582459400f35)
![Screenshot 2024-07-07 at 14 47 30](https://github.com/orhun/gpg-tui/assets/368260/3e12bcfd-7861-40ff-b80c-29c4abeaa49c)

**Comment by orhun** [comment]
Hey, can you give more information about what this is about? 🤔

**Reply by bsodmike** [reply]
https://github.com/orhun/gpg-tui/pull/246

**Comment by bsodmike** [comment]
Will close for us to discuss further in the PR 😊
